monero-outreach-docs/translations/es/PSA/files/mining/mine_support_network_es.md

```diff
@@ -0,0 +1,27 @@
+# Minar Monero para apoyar la red
+
+Los mineros de Monero utilizan hardware informático para ejecutar una variante del algoritmo Cryptonight y competir por una recompensa asociado al siguiente bloque en la blockchain. La minería de Monero es reconocida por su resistencia a los ASIC. Esto quiere decir que es más accesible y amigable minar con CPU (unidad central de proceso) y hardware GPU (unidad de procesamiento gráfico), pero no garantiza la configuración que realizan los individuos y las compañias a los poderosos mineros ASICs y los FPGAs para minar Monero. Los desarrolladores de Monero han establecido una actualización rutinaria sobre la red, que tiene lugar cada 6 meses que ayuda a combatir en contra de la minería de Monero a través de los ASICs y los FPGAs y evitar que estos puedan obtener el control de la red a través del hashrate (poder de procesamiento).
+
+Sin embargo, se ha asumido que el valor intrínseco o la rentabilidad general de la minería de Monero empieza a ser notable, más individuos y empresas intentarán configurar los ASICs y los FPGAs para tener una ventaja comparativa y llevarse las recompensas de la minería en contra de los mineros CPU/GPU con un hashrate menor. Algunos miembros de la comunidad les preocupa el riesgo sobre un ataque del 51% en Monero de parte de los ASICs y los FPGAs con su alto poder de procesamiento hash que sus máquinas pueden ofrecer.
```

Comment:
Other suggestions:

- _...intentarán  configurar **dispositivos ASIC y FPGA**... ...mineros CPU/GPU con un **poder de procesamiento** menor._ If I recall, we selected that translation, and also _poder de hash_.
- _Algunos miembros de la comunidad se preocupan sobre el riesgo de los dispositivos ASIC y FPGA presentando un ataque del 51% en Monero con su alto poder de procesamiento que sus máquinas pueden ofrecer._

Reply:
done, thanks for the reminder, sometimes I forget to check the terminology. :)